src/styles/card.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  padding: 32px 16px 16px;
+
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+
+  margin-bottom: 40px;
+
+  display: block;
+}
+
+.card__product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+
+  margin-bottom: 16px;
+}
+
+.card__review-wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  margin-bottom: 24px;
+}
+
+.card__review-value {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 16px;
+}
+
+.card__price-title {
+  color: #616070;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+.card__price-value {
+  color: #060b35;
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card__buy-bottom {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+
+  transition: background-color 0.3s, color 0.3s;
+}
+
+.card__buy-bottom:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
```

Comment:
Check the p.9 of the [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

Reply:
ok, I will change